topsort-api-v2.yml

```diff
@@ -1158,6 +1160,10 @@ components:
         title:
           type: string
           description: An optional title for the sponsored brand selected when creating the campaign.
+        vendorId:
+          type: string
+          description: The ID of the vendor associated with this sponsored brand winner.
+          example: v_8fj2D
```

Comment:
This is optional, mark it as such. Perhaps mention that it is not enabled by efault